src/FlaUI.WebDriver.UITests/SessionTests.cs

```diff
@@ -321,10 +323,15 @@ public void NewCommandTimeout_NotExpired_DoesNotEndSession()
         public void NewCommandTimeout_InvalidValue_Throws(object value)
         {
             var driverOptions = FlaUIDriverOptions.TestApp();
-            driverOptions.AddAdditionalOption("appium:newCommandTimeout", value);
+            driverOptions.AddAdditionalAppiumOption("appium:newCommandTimeout", value);
 
-            Assert.That(() => new RemoteWebDriver(WebDriverFixture.WebDriverUrl, driverOptions), 
+            Assert.That(() => new WindowsDriver(WebDriverFixture.WebDriverUrl, driverOptions), 
                 Throws.TypeOf<WebDriverArgumentException>().With.Message.EqualTo("Capability appium:newCommandTimeout must be a number"));
         }
+
+        private class TestOptions : DriverOptions
```

Comment:
This is needed because `AppiumOptions`' capabilities are typed, and so it wasn't possible to test mistyped capabilities with it.